python/paddle/fluid/layers/nn.py

```diff
@@ -3759,3 +3760,44 @@ def label_smooth(label,
         outputs={"Out": smooth_label},
         attrs={"epsilon": float(epsilon)})
     return smooth_label
+
+
+def roi_pool(input, rois, pooled_height=1, pooled_width=1, spatial_scale=1.0):
```

Comment:
Please also add `roi_pool` in doc : https://github.com/PaddlePaddle/Paddle/blob/develop/doc/fluid/api/layers.rst#nn